### ISSUE TITLE
How to compress model in Multi-machine 

### ISSUE THREAD

**Issue by as754770178** [Tencent/PocketFlow#35]
I want to compress the model in How to compress model in Multi-machine

**Comment by jiaxiang-wu** [comment]
You mean how to train a compressed model with multiple nodes/machines?

**Comment by xieydd** [comment]
When using the horovod, i think it`s feasible.

**Comment by as754770178** [comment]
Yes, but I mean compress the model by Prunning/Quantization, then train the compress model with multiple nodes/machines. 

I have other questions:
1. Can PocketFlow compress Classification network， such as Yolo, fasterCNN?
2. I how to verify the inference speed of compressed model is faster than the full-precision uncompressed network?

thanks

**Comment by as754770178** [comment]
> When using the horovod, i think it`s feasible.

What's it?

**Comment by xieydd** [comment]
Horovod is Distributed training framework for TensorFlow, Keras, and PyTorch.
1.  AFAIK，it not yet support the state-of-art models lke the Yolo or fasterCNN
2. you can test in TensorRT

**Comment by as754770178** [comment]
I mean use tf-serving to test the speed? Can horovod support compress model? I want compress model and train it together, like PocketFlow.

**Comment by jiaxiang-wu** [comment]
1. Horovod is a distributed training framework. It does not care what code is running within.
2. The speed improvement on GPU will be less significant than on ARM-based mobile devices, according to our experience.

**Comment by as754770178** [comment]
Can PocketFlow compress multi-label Classification network?

**Comment by jiaxiang-wu** [comment]
Yes, you can define a new `ModelHelper` class, including the network architecture and the computation of loss function's value, similar to other `ModelHelper` classes defined under the `nets` directory.

**Comment by as754770178** [comment]
OK, thanks.
